documentation/smg/14_documentation.md

```diff
@@ -0,0 +1,45 @@
+# Documentation
+
+This document will outline the process surrounding the documentation currently housed in the `horace-docs` repository. It assumes a basic knowledge of Restructured Text (ReST) format (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) especially the subset of the sphinx-doc toolset (https://www.sphinx-doc.org/en/master/)
+
+## Documentation style guide
+
+
+## Adding new documents
+### Create new `.rst`
+
+The first job to add a new document is to create it. 
+
+* The file should be added into the `docs` folder, preferably with the `.rst` suffix. 
+   * For ease the filename of the document should be the title with spaces replaced by underscores. 
+* It should be written using the sphinx-doc style, including interfile references through:
+
+    ```
+    :ref:`link-label <link_file:link_location>`
+    ```
+    
+ * External references are written in the usual ReST format:
+
+    ```
+    `link-label <https:\\link-to-elsewhere>`
+    ```
+
+### Update index
+
+The main file in the docs folder is `index.rst` which consists of two major parts. 
+
+#### The TOCTree
+
+The TOC tree describes all child rst files to be loaded for the purpose of references. It also provides the the details for the links sidebar. 
+
+To include a document and include it in the side bar, simply add the new filename (without file extension) to the toctree. 
+
+To include a document for reference, but not in the sidebar
```

Comment:
Presuming this line "To include... not in sidebar" refers to the next section for the main index, and I think it should preferably go after the heading. If you think it important that it should go before the heading, then please add an ellipsis ... to indicate that the meaning is continued in the following text.

Reply:
Sorry, that was a trailing bit that I had left to fill in when I had decided on a syntax.